cmd/apg/migrate-database.go

```diff
@@ -0,0 +1,143 @@
+// Code generated. DO NOT EDIT.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"fmt"
+
+	"github.com/golang/protobuf/jsonpb"
+
+	"os"
+
+	rpcpb "github.com/apigee/registry/rpc"
+)
+
+var MigrateDatabaseInput rpcpb.MigrateDatabaseRequest
+
+var MigrateDatabaseFromFile string
+
+var MigrateDatabaseFollow bool
+
+var MigrateDatabasePollOperation string
+
+func init() {
+	AdminServiceCmd.AddCommand(MigrateDatabaseCmd)
+
+	MigrateDatabaseCmd.Flags().StringVar(&MigrateDatabaseInput.Kind, "kind", "", "A string describing the kind of migration to...")
+
+	MigrateDatabaseCmd.Flags().StringVar(&MigrateDatabaseFromFile, "from_file", "", "Absolute path to JSON file containing request payload")
+
+	MigrateDatabaseCmd.Flags().BoolVar(&MigrateDatabaseFollow, "follow", false, "Block until the long running operation completes")
```

Comment:
Within g3 LRO, we usually use `wait` to wait until LRO finish

Reply:
SGTM - this is code generated by another project, so not something we can address here.

Reply:
Ack. Would this be addressed separately or we'll use `--follow` for a while? I'm thinking when would be a good time to update data plane's provisioning template

Reply:
If you're using the `apg` tool in provisioning, I think using `--follow` would be much less overhead than changing the CLI generator. Also, this always returns completed operations and the PR doesn't include an implementation of the LRO service - we'll want to add that eventually.